howso/feature_residuals.amlg

```diff
@@ -234,6 +234,8 @@
 	; confusion_matrix_min_count: number, optional, default is 10. Applicable only to confusion matrices, the number of predictions a class should have
 	;		(value of a cell in the matrix) for it to remain in the confusion matrix. If the count is less than this value, it will be accumulated
 	;		into a single value of all insignificant predictions for the class and removed from the confusion matrix.
+	; use_shared_deviations: flag, optional. If set to true, will return the residuals grouped by shared deviations group, otherwise will return the residuals
```

Comment:
I feel like we'd be better off having this default to False and only turn it on within analyze or something